libs/designsystem/src/lib/components/radio/radio-group/radio-group.component.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  AfterContentInit,
+  ContentChild,
+  ContentChildren,
+  EventEmitter,
+  Output,
+  QueryList,
+  TemplateRef,
+  ViewChildren,
+} from '@angular/core';
+import { Component, Input } from '@angular/core';
+
+import { ListItemTemplateDirective } from '../../list/list.directive';
+import { RadioComponent } from '../radio.component';
+
+@Component({
+  selector: 'kirby-radio-group',
+  templateUrl: './radio-group.component.html',
+  styles: ['ion-radio-group { display: inherit; flex-wrap: inherit}'],
+})
+export class RadioGroupComponent implements AfterContentInit {
```

Comment:
Yes it should, I've deliberately kept it out of the 1st version to avoid scope creep and get something shipped :)
I'll update the PR description and create an enhancement issue for ngForms 👍